util/conversion/conversion.go

```diff
@@ -30,13 +35,63 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/util/json"
 	"k8s.io/utils/diff"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/cluster-api/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/conversion"
 )
 
 const (
 	DataAnnotation = "cluster.x-k8s.io/conversion-data"
 )
 
+var (
```

Comment:
```suggestion
const (
```

Reply:
It can't actually be assigned to a constant because the value comes from a function and can only be resolved at runtime